src/main/java/jnr/posix/LinuxFileStat64.java

```diff
@@ -10,21 +10,22 @@ public Layout(jnr.ffi.Runtime runtime) {
         }
 
         public final Signed64 st_dev = new Signed64();
-        public final Signed64 st_ino = new Signed64();
+        public final Unsigned64 st_ino = new Unsigned64();
         public final Signed64 st_nlink = new Signed64();
-        public final Signed32 st_mode = new Signed32();
-        public final Signed32 st_uid = new Signed32();
-        public final Signed32 st_gid = new Signed32();
+        public final Unsigned32 st_mode = new Unsigned32();
+        public final Unsigned32 st_uid = new Unsigned32();
+        public final Unsigned32 st_gid = new Unsigned32();
+        public final Signed32 __pad0 = new Signed32();
```

Comment:
@etehtsea Is __pad0 missing from all 64 bit Linux systems?  This seems like everything would be way off here if this were the case.  Assuming it was ever tested against any other 64 bit linux.